bundler.go

```diff
@@ -360,7 +365,7 @@ func (b *Bundler) bundle(e ConfigurationEnvironment) (err error) {
 			`"main.BuiltAt=` + time.Now().String() + `"`,
 		},
 	}
-	if e.OS == "windows" {
+	if e.OS == "windows" && b.KeepWindowsConsole != true {
```

Comment:
Is there a reason why you don't use `!b.KeepWindowsConsole` instead of `b.KeepWindowsConsole != true` since `b.KeepWindowsConsole` is a `bool`?

Reply:
Hi! Not have any reasons, just learning the language and still getting confused :)

Reply:
OK could you replace it with `!b.KeepWindowsConsole` then ?